content/chapters/compute/lab/support/sum-array-bugs/seg-fault/generate_random_array.c

```diff
@@ -0,0 +1,14 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
```

Comment:
```suggestion
// SPDX-License-Identifier: BSD-3-Clause
```
https://www.kernel.org/doc/html/v5.0/process/license-rules.html#license-identifier-syntax